index.js

```diff
@@ -57,6 +57,10 @@ Benchmarked.prototype.defaults = function(benchmarked) {
     }
   };
 
+  this.format = function(benchmark) {
+    return '  ' + benchmark;
+  };
```

Comment:
maybe we should just do that outside of defaults and make it a prototype method or something?